keychain_unix.cpp

```diff
@@ -91,13 +99,18 @@ static KeyringBackend detectKeyringBackend()
     case DesktopEnv_Plasma5:
         return Backend_Kwallet5;
```

Comment:
Could we check somehow if kwallet5 is running.  And if it is not, then we'd fallback to libsecret?

Reply:
Will check.